app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.ts

```diff
@@ -470,6 +470,7 @@ export class AwsLoadBalancerTransformer {
               deregistrationDelayConnectionTermination: Boolean(
                 targetGroup.attributes['deregistration_delay.connection_termination.enabled'] === 'true',
               ),
+              preserveClientIp: Boolean(targetGroup.attributes['preserve_client_ip.enabled'] === 'true'),
```

Comment:
lgtm except I'm confused why we use `attributes['preserve_client_ip.enabled']` vs `targetGroup.attributes.preserveClientIp`